### ISSUE TITLE
Error creating 2-of-3 multisig wallet

### ISSUE THREAD

**Issue by mflaxman** [cryptoadvance/specter-desktop#287]
Could be because the node I'm using isn't fully sync-ed, it appears to be timing out:
![Screen Shot 2020-08-17 at 3 00 28 PM](https://user-images.githubusercontent.com/448644/90438995-6efed000-e09a-11ea-97a4-82e6b2011d40.png)

http://127.0.0.1:25441/new_wallet/multisig/
```
Traceback (most recent call last):
  File "/private/tmp/.venv3/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/private/tmp/.venv3/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/private/tmp/.venv3/lib/python3.7/site-packages/flask_login/utils.py", line 269, in decorated_view
    return func(*args, **kwargs)
  File "/private/tmp/.venv3/lib/python3.7/site-packages/cryptoadvance/specter/controller.py", line 843, in new_wallet
    wallet = app.specter.wallet_manager.create_wallet(wallet_name, sigs_required, address_type, keys, cosigners)
  File "/private/tmp/.venv3/lib/python3.7/site-packages/cryptoadvance/specter/wallet_manager.py", line 240, in create_wallet
    return self.wallets[name]
KeyError: 'Multisig'
```

I'm not sure what else it might be, this worked previously. But I figured regardless it should throw a better error.

**Comment by mflaxman** [comment]
I was running an old v 0.18 node, this error message disappears on a v 0.20.0 node.

I'm fine to close this issue, unless you want to leave it open to address the bad error message.

**Comment by ben-kaufman** [comment]
Thanks, this actually might be the same issue as with #284
This definitely shouldn't be the error message, I'll make sure we handle that case properly!

**Comment by mflaxman** [comment]
What solved this for me:
```
$ rm -rf ~/.specter/
```

**Comment by ben-kaufman** [comment]
Thanks for checking that :) so I'm guessing something is wrong probably with permissions... will check that.

**Comment by stepansnigirev** [comment]
Solved in v0.7.1
